de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/api/impl/ReceiptScheduleBL.java

```diff
@@ -205,16 +209,14 @@ public I_C_BPartner getC_BPartner_Effective(final I_M_ReceiptSchedule rs)
 		return bpartnersRepo.getById(bpartnerId, I_C_BPartner.class);
 	}
 
+	@Nullable
 	@Override
-	public int getAD_User_Effective_ID(final I_M_ReceiptSchedule rs)
+	public BPartnerContactId getBPartnerContactID(@NonNull final I_M_ReceiptSchedule rs)
 	{
-		final int adUserOverrideID = rs.getAD_User_Override_ID();
-		if (adUserOverrideID > 0)
-		{
-			return adUserOverrideID;
-		}
+		final int cBPartnerIdRepo = rs.getC_BPartner_Override_ID() > 0 ? rs.getC_BPartner_Override_ID() : rs.getC_BPartner_ID();
```

Comment:
that's OK, but next time, please use `CoalesceUtil.firstGreaterThanZero(int...)`